vipps-ecom-api.md

```diff
@@ -1076,6 +1077,18 @@ The user's phone number can be set in the payment initiation call:
 The user's phone number is remembered by the user's browser,
 eliminating the need for re-typing it on subsequent purchases.
 
+### Generating a QR code to the Vipps landing page
+
+You can generate a short url or a QR code to take the user to the Vipps landing page for a one-time payment.
```

Comment:
You can replace this with:

"You can generate a QR based on the landing page url, if you have user-facing screen (instead of having to ask a customer for their phonenumber). Scanning the QR will take the user directly to the payment in the app."

Since we dont want to promote the shortening, we want to promote the QR :)

Reply:
Great! :-)